unit-tests/DispatcherCliTest.php

```diff
@@ -95,4 +95,27 @@ public function testDispatcher()
 
 	}
 
+	public function testCliParameters()
+	{
+		$di = new \Phalcon\DI\FactoryDefault\CLI();
+
+		$dispatcher = new \Phalcon\CLI\Dispatcher();
+
+		$di->setShared("dispatcher", $dispatcher);
+
+		$dispatcher->setDI($di);
+
+		$dispatcher->setTaskName('params');
+		$dispatcher->setActionName('params');
+		$dispatcher->setParams(array('This', 'Is', 'An', 'Example'));
+		$dispatcher->dispatch();
+		$this->assertEquals($dispatcher->getReturnedValue(), 'same');
+
+		$dispatcher->setTaskName('params');
+		$dispatcher->setActionName('param');
+		$dispatcher->setParams(array('This', 'Is', 'An', 'Example'));
+		$dispatcher->dispatch();
+		$this->assertEquals($dispatcher->getReturnedValue(), 'same');
```

Comment:
Can you please explain why two identical test case? Also can you please create stress-test for `\Phalcon\Cli\Dispatcher::callActionMethod` and  `\Phalcon\Dispatcher::callActionMethod` because they're public